src/replication.c

```diff
@@ -192,6 +192,9 @@ replicator(void *arg) {
 			MTX_UNLOCK(&replica->r_mtx);
 			if((rcmd->opcode == ZVOL_OPCODE_READ) && read_cmd_sent) {
 				break;
+			} else if(rcmd->opcode == ZVOL_OPCODE_WRITE) {
+				//TODO This needs to be added in the starting
+				cmd->copies_sent++;
```

Comment:
we can do this for read IOs also.. and do this right after inserting into sendq of replica